tt_metal/impl/dispatch/kernels/remote_issue_queue_interface.cpp

```diff
@@ -0,0 +1,95 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
Why are we calling the kernel an interface?

Reply:
ah I named it this way because its the kernel that reads from issue queue, would you prefer `remote_issue_queue_fetcher`?